python/lsst/daf/butler/cli/opt/options.py

```diff
@@ -127,8 +128,12 @@ def makeCollectionTypes(context, param, value):
         normalize=True,
         unseparated_okay=True,
         add_to_default=True,
+        default_key=None,  # No separator
     ),
-    help=f"The logging level. Supported levels are [{'|'.join(logLevelChoices)}]",
+    help=f"The logging level. Without an explicit logger name, will only affect the default root loggers "
+    f"({', '.join(CliLog.root_loggers())}). "
+    f"To modify the root logger set '__root__=LEVEL' or '=LEVEL'. "
+    f"Supported levels are [{'|'.join(logLevelChoices)}]",
```

Comment:
Why not add `TRACE` to this list?

Reply:
I could. I only thought about it when I was writing the review request on Jira.